packages/cli/src/commands/ui/create/vue.ts

```diff
@@ -143,7 +143,7 @@ export default class Vue extends Command {
         //   user: providerUsername,
         // },
       },
-      cssPreprocessor: 'node-sass',
+      cssPreprocessor: 'sass',
```

Comment:
Node Sass causes some errors on the pnpm installation.
Node Sass is deprecated anyway.